app/views/hyrax/base/_form_share.html.erb

```diff
@@ -62,10 +62,21 @@
 <h2>Share work with groups of users</h2>
 <p class="sr-only">Use the add button to give access to one group at a time (it will be added to the list below).</p>
 <div class="form-group row">
+  <!--<div class="col-sm-5">-->
+    <!--<label for="new_group_name_skel" class="sr-only">Group</label>-->
+    <!--<%=# select_tag 'new_group_name_skel', options_for_select(["Select a group"] + current_user.groups), class: 'form-control' %>-->
+  <!--</div>-->
+
   <div class="col-sm-5">
+    <%
```

Comment:
The point of the two lines of code was to be clear that the admin-group was only excluded for an admin user, this seemed to be the minimal change that could be made.  I'm not yet clear about all the details on how the admin-group is defined and to be used (additional questions below).  If the one-liner is used - the logic changes - it excludes the admin-group from sharing, period.  I need to clarify whether non-admin users should be able to share content with an admin-group and, if so, that would imply that the sharing groups could include groups that a user does not belong to.